tools/nimgrep.nim

```diff
@@ -50,44 +58,132 @@ Options:
                -b:N   print N lines of leading context before every match
   --context:N, -c:N   print N lines of leading context before every match and
                       N lines of trailing context after it
+  --sortTime          order files by the last modification time -
+       -s[:asc|desc]  - ascending (default: recent files go last) or descending
   --group, -g         group matches by file
   --newLine, -l       display every matching line starting from a new line
+  --limit[:N], -m[:N] limit max width of lines from files by N characters (80)
+  --fit               calculate --limit from terminal width for every line
+  --onlyAscii, -o     use only printable ASCII Latin characters 0x20-0x7E
```

Comment:
remove `-o` shortcut, it's too often associated with output file (again, users can alias their favorite commands)

Reply:
Agree. However I really need such shortcuts, so I added -@. I hope it's not too weird.